app/web/features/dashboard/Hero/HeroButton.tsx

```diff
@@ -8,11 +8,6 @@ import makeStyles from "utils/makeStyles";
 
 import useHeroBackgroundTheme from "./useHeroBackgroundTheme";
 
-declare module "@mui/styles/defaultTheme" {
-  // eslint-disable-next-line @typescript-eslint/no-empty-interface
-  interface DefaultTheme extends Theme {}
-}
-
```

Comment:
I think I added these out of anxiety during the first MUI upgrade but they don't see to be needed. At least all the Hero stuff looks fine.

The important one of these is the one in the theme file, which is still there.